drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/validator/PatternBindingValidator.java

```diff
@@ -0,0 +1,10 @@
+package org.drools.workbench.screens.guided.rule.client.editor.validator;
+
+import java.util.Collection;
+
+import org.guvnor.common.services.shared.validation.model.ValidationMessage;
+
+public interface PatternBindingValidator {
```

Comment:
You also use implementations of this ```interface``` for **field** binding validation; so perhaps the name could be improved.